src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractClass<Item> implements ItemDao {
+    private static final String DB_NAME = "test";
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.DB_NAME);
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) {
+        String name = item.getName();
+        BigDecimal price = item.getPrice();
+        String query = "INSERT INTO " + DB_NAME + ".items (name, price) "
+                + "VALUES('" + name + "', " + price + ")";
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(query);
+        } catch (SQLException e) {
+            logger.error("Cannot create item\n" + e);
```

Comment:
```suggestion
            logger.error("Cannot create item", e);
```
optional, but I prefer this :)

Reply:
done